### PR TITLE
Allow hiding backup tokens in the backup token view

### LINKED ISSUE #34: Should backup codes be viewable more than once?
Currently backup codes are shown every time you go to the 2FA settings page. Most "big" websites will only show these to you once (but will let you completely regenerate them).

I'm unsure if there's a "best practice" here.

### PR DESCRIPTION
**Pull request by akx**
Extracted from #69.

Fixes #34.

### CODE REVIEW

File: allauth_2fa/app_settings.py [hunk 1]
```diff
@@ -5,3 +5,5 @@
 TEMPLATE_EXTENSION = getattr(settings,
                              'ALLAUTH_2FA_TEMPLATE_EXTENSION',
                              allauth_settings.TEMPLATE_EXTENSION)
+
+ALWAYS_REVEAL_BACKUP_TOKENS = bool(getattr(settings, 'ALLAUTH_2FA_ALWAYS_REVEAL_BACKUP_TOKENS', True))
```
Comment (clokep) [inline allauth_2fa/app_settings.py:9]: I don't think this `bool` cast will generally do what is expected with strings. Is this necessary here? (E.g. `bool('False') == True`)

File: allauth_2fa/views.py [hunk 2]
```diff
@@ -163,6 +163,9 @@ def get_form_kwargs(self):
 
 class TwoFactorBackupTokens(TemplateView):
     template_name = 'allauth_2fa/backup_tokens.' + app_settings.TEMPLATE_EXTENSION
+    # This can be overridden in a subclass to True,
+    # to have that particular view always reveal the tokens.
+    reveal_tokens = bool(app_settings.ALWAYS_REVEAL_BACKUP_TOKENS)
```
Comment (clokep) [inline allauth_2fa/views.py:168]: Why the additional cast to `bool` here?

**Review by clokep — COMMENTED** [review]
Overall looks good, but I have a couple questions about the bool casts.

File: allauth_2fa/app_settings.py (hunk 1 above)
Comment (akx) [reply allauth_2fa/app_settings.py:9]: It's there as a type hint, really. It won't do the right thing for most things for sure, but at least the user/reader will know it's expected to be a bool.

File: allauth_2fa/views.py (hunk 2 above)
Comment (akx) [reply allauth_2fa/views.py:168]: Another type hint (for IDEs and such).

File: allauth_2fa/app_settings.py (hunk 1 above)
Comment (clokep) [reply allauth_2fa/app_settings.py:9]: OK. It probably isn't any worse than the implicit conversion that will happen when this is used in the template anyway.

**Pull request merged**